posthog/hogql/database/schema/util/where_clause_extractor.py

```diff
@@ -329,6 +333,26 @@ def handle_timestamp_comparison(
                         right=node.left,
                     )
                 )
+
+        if node.op == CompareOperationOp.Eq:
+            if is_left_constant and is_session_id_string_expr(node.right, self.context):
+                left_timestamp_expr = self.session_id_str_to_timestamp_expr(node.left)
```

Comment:
It is possible for `session_id_str_to_timestamp_expr` to return None? If so, how will be known about it?